lib/run/index.js

```diff
@@ -336,6 +344,10 @@ module.exports = function (options, callback) {
                 if (!(_.isFunction(Reporter) && _.isObject(emitter.reporters[reporterName]))) {
                     // @todo: route this via print module to respect silent flags
                     console.warn(`newman warning: "${reporterName}" reporter could not be loaded.`);
+                    // print install instruction in case a valid custom reporter is missing
+                    if (_.includes(validCustomReporters, reporterName)) {
+                        console.warn(`  run 'npm install newman-reporter-${reporterName}' to install`);
```

Comment:
This is a very authoritative message and as such, the messaging needs to be changed. For `@namespace/newman-reporter-reportername`, this would be misleading.

Kindly store message in object and use from there and wherever preset message is not found, add a message saying "please install reporter using npm".